src/utils/morpho.ts

```
@@ -22,6 +42,8 @@ export const getIRMTitle = (address: string) => {
       return 'Adaptive Curve';
     case '0x46415998764c29ab2a25cbea6254146d50d22687': // on base
       return 'Adaptive Curve';
+    case '0xe675A2161D4a6E2de2eeD70ac98EEBf257FBF0B0': // on polygon
```

Biome (1.9.4)

[error] This expression always returns false, because the string is converted and will never match

This call convert the string to lower case

... but this value is not in lower case

Unsafe fix: Use lower case string value.


(lint/correctness/noStringCaseMismatch)